index.html

```diff
@@ -5,8 +5,12 @@
     <script>
 
       // This script makes an API call and displays the returned data
+
```

Comment:
@chaneberg didn't you already merge this into dev? I don't know why this is showing up in the diff.